client/pages/game.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+export default class Game extends React.Component {
+
+  render() {
+    const { title, img, deck } = this.props.game;
+    return (
+      <div className="game-container" id="game-container">
+        <div className="row">
+          <div className="back-arrow">
+          <a href="#search">
+          <i className="fas fa-arrow-left arrow-icon" ></i>
+          </a>
+        </div>
+        </div>
+        <div className="row">
+          <div className="img-cont">
+            <img src={img}></img>
```

Comment:
we should try forcing them to be langscape. it probably won't look as bad. You have to send the image height and width, but then use object-fit in the CSS. i think it will look WAY better event if some of the image is cut off